src/lib/packager/clients/repositories.rb

```diff
@@ -362,6 +366,8 @@ def repoInfoRichText(name, category, info)
         "\">&nbsp;&nbsp;&nbsp;"
       )
 
+      name_string = name != raw_name ? _("Raw name: %s") % raw_name + "<BR>" : ""
```

Comment:
I think we should add a comment for translators so they know the context. Something like

```ruby
# TRANSLATORS: the raw name is the original name with unexpanded variables like "$releasever"
```